docs/CODERABBIT_AUTO_REVIEW.md

```
@@ -0,0 +1,416 @@
+---
+
+## 📖 상세 단계
+
+### Step 1: 스크립트 실행
+
+```bash
+./scripts/commands/apply-coderabbit-review.sh 3
+```
+
+**출력 예시:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)